packages/node_modules/@ciscospark/internal-plugin-presence/test/integration/spec/presence.js

```diff
@@ -81,6 +93,32 @@ describe(`plugin-presence`, function() {
             assert.equal(presenceResponse.responses[1].subject, spock.id);
           })
       );
+
+      it(`should receive a mercury event for a subscribed person's change`, () => {
+        const presenceUpdateSpy = sinon.spy();
+        spock.spark.internal.mercury.on(`event:apheleia.subscription_update`, presenceUpdateSpy);
```

Comment:
Is the spy doing anything that waiting for the event doesn't?

Reply:
oh, i see, this is where you're getting the value of the subscription event. `eventEvent()` should give you the event as well, so that might be an easier place to grab the property from.